compiler/test/formatter_inputs/includes.gr

```diff
@@ -28,7 +28,7 @@ include "runtime/unsafe/wasmi32"
 from WasmI32 use {
   eq, // comment1
   // comment 3
-  and as (&), // comment 2
+  and_ as (&), // comment 2
```

Comment:
sadge

Reply:
I think this is a mistake that I just forgot to rectify when I rebased my branch into main with `WasmI32` stuff being operators. I probably just missed it here since it didn't raise a compile error because it's just a formatter input